### PR TITLE
Garrin loan model and listing

### PR DESCRIPTION
**Pull request by JialinWang1**

### CODE REVIEW

File: front-end/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.15.1",
     "jest-expo": "^35.0.0",
-    "react-test-renderer": "^16.12.0",
+    "react-test-renderer": "16.8",
```
Comment (adamgyulavari) [inline front-end/package.json]: @JialinWang1 why is this change here?

Comment (JialinWang1) [reply front-end/package.json]: probably because of the merging conflicts, i had accepted the souce changes, i had now changed it to "react-test-renderer": "^16.12.0"

File: back-end/tests/service_test/loansService.test.js [hunk 2]
```diff
@@ -3,12 +3,9 @@ const db = require('../../services/dbService')
 
 // mocking the hold dbService
```
Comment (adamgyulavari) [inline back-end/tests/service_test/loansService.test.js]: @JialinWang1 this is still here :)

File: back-end/index.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
 require('dotenv').config()
-const { init, selectByUserId } = require('./services/dbService')
+const { init } = require('./services/dbService')
 const app = require('./app')
 
 const PORT = process.env.PORT || 3000
 
 init()
```
Comment (adamgyulavari) [inline back-end/index.js]: @JialinWang1 still not obvious what is getting initialized
